skyportal/models.py

```diff
@@ -206,6 +208,71 @@ def desi_dr8_url(self):
         return (f"http://legacysurvey.org/viewer/jpeg-cutout?ra={self.ra}"
                 f"&dec={self.dec}&size=200&layer=dr8&pixscale=0.262&bands=grz")
 
+    def airmass(self, telescope, time, below_horizon=np.inf):
+        """Return the airmass of the Obj at time `time` from Telescope
```

Comment:
We typically use a sentence without any parameters here, e.g.

"Return the airmass of an object at a given time."